acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/steps/LoginSteps.java

```diff
@@ -0,0 +1,54 @@
+package uk.nhs.digital.ps.test.acceptance.steps;
+
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.springframework.beans.factory.annotation.Autowired;
+import uk.nhs.digital.ps.test.acceptance.pages.DashboardPage;
+import uk.nhs.digital.ps.test.acceptance.pages.LoginPage;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+public class LoginSteps extends AbstractSpringSteps {
+
+    @Autowired
+    private LoginPage loginPage;
+
+    @Autowired
+    private DashboardPage dashboardPage;
+
+    @Given("^I am on login page$")
+    public void iAmOnLoginPage() throws Throwable {
+        loginPage.open();
+    }
+
+    @When("^I submit my valid admin credentials$")
+    public void iSubmitMyValidAdminCredentials() throws Throwable {
+        loginPage.loginWith("admin", "admin");
+    }
+
+    @Then("^a dashboard is displayed$")
+    public void aDashboardIsDisplayed() throws Throwable {
+        assertThat("Current page is the dashboard page.", dashboardPage.isOpen(), is(true));
+    }
+
+    @When("^I submit my invalid credentials$")
+    public void iSubmitMyInvalidCredentials() throws Throwable {
+        loginPage.loginWith("admin","wrongpassword");
+    }
+
+    @Then("^an error is displayed$")
+    public void anErrorIsDisplayed() throws Throwable {
+        // @formatter:off
```

Comment:
Do we really need to disable formatting?

Reply:
I thought this long statement reads better when broken down like this and therefore wanted to ensure it remains that way in case someone runs auto-formatter against the whole file.

Having said that, I'm not fussed about it here - the line is simple enough that even when it breaks down after the second comma (which happens when default auto-formatting is applied) it still remains sufficiently readable. I'm removing the auto-formatter statements.